### ISSUE TITLE
Blues scale is wrong in Quantizer

### ISSUE THREAD

**Issue by jhoar** [jeremywen/JW-Modules#8]
The definition of the Blues scale here: 

http://www.grantmuller.com/MidiReference/doc/midiReference/ScaleReference.html 

is wrong, should be 

1, b3, 4,  b5, 5, b7 
(0, 3, 5, 6, 7, 10)

**Comment by jeremywen** [comment]
Thank You.  Will fix that and the other issue you found.  😄

**Comment by jeremywen** [comment]
I have fixed some issues listed here https://github.com/jeremywen/JW-Modules/releases
I decided not to scale to the next octave.  Maybe in the future I will add more options.  Ableton live's midi scale effect always 'rounds' *down* to the highest note in the scale instead of up to the next octave.  I think this is just a matter of preference.  Pfft I don't know.  After two days of working on the quantizer I just hope I am mostly correct 😄 Let me know what you think.  Thanks for your help!
